arrows/ffmpeg/ffmpeg_video_input.cxx

```diff
@@ -446,40 +482,43 @@ class ffmpeg_video_input::priv
       }
 
       // grab the metadata from this packet if from the metadata stream
-      auto md_iter = this->metadata.find(this->f_packet->stream_index);
-      if (md_iter != this->metadata.end() )
+      auto md_iter = this->metadata.find( this->f_packet->stream_index );
+      if( md_iter != this->metadata.end() )
       {
         md_iter->second.emplace(
-          this->f_packet->pts,
-          std::deque<uint8_t>(this->f_packet->data,
-                              this->f_packet->data + this->f_packet->size));
+            this->f_packet->pts,
+            std::deque< uint8_t >( this->f_packet->data,
+                                   this->f_packet->data +
+                                   this->f_packet->size ) );
       }
 
       // De-reference previous packet
-      av_packet_unref(this->f_packet);
+      av_packet_unref( this->f_packet );
     }
 
     // The cached frame is out of date, whether we managed to get a new
     // frame or not.
     this->current_image_memory = nullptr;
 
-    if (!this->frame_advanced)
+    if( !this->frame_advanced )
     {
-      this->f_frame->data[0] = NULL;
+      this->f_frame->data[ 0 ] = NULL;
     }
 
-    for (auto& md : this->metadata)
+    for( auto& md : this->metadata )
     {
-      for (auto md_it = md.second.begin(); md_it != md.second.end(); )
+      for( auto md_it = md.second.begin(); md_it != md.second.end();)
       {
         // Skip if timestamp is too large while in sync mode
-        if (md_it->first <= this->f_pts || !this->sync_metadata)
+        if( md_it->first <= this->f_pts || !this->sync_metadata )
         {
-          this->curr_metadata[md.first].insert(curr_metadata[md.first].end(),
-            md_it->second.begin(), md_it->second.end() );
+          this->curr_metadata[ md.first ].insert(
+               curr_metadata[ md.first ].end(),
+              md_it->second.begin(),
+              md_it->second.end() );
```

Comment:
How did *this* happen, I wonder? Unless I am reading this wrong somehow, it should be:
```suggestion
          this->curr_metadata[ md.first ].insert(
            curr_metadata[ md.first ].end(),
            md_it->second.begin(),
            md_it->second.end() );
```